src/nrc/api/tests/test_abonnement.py

```diff
@@ -303,3 +303,75 @@ def test_abonnement_destroy_different_clientid(self):
         response = self.client.delete(abonnement_url)
 
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_abonnementen_create_duplicate_callbacks(self):
+        """
+        test /abonnementen POST:
+        create abonnement with nested kanalen and nested filters via POST request
+        check if data were parsed to models correctly
+        """
+        KanaalFactory.create(
+            naam="zaken", filters=["bron", "zaaktype", "vertrouwelijkheidaanduiding"]
+        )
+        KanaalFactory.create(naam="informatieobjecten", filters=[])
+        abonnement_create_url = get_operation_url("abonnement_create")
+
+        data = {
+            "callbackUrl": "https://ref.tst.vng.cloud/zrc/api/v1/callbacks",
+            "auth": "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiIsImNsaWVudF9pZG"
+            "VudGlmaWVyIjoienJjIn0.eyJpc3MiOiJ6cmMiLCJpYXQiOjE1NTI5OTM"
+            "4MjcsInpkcyI6eyJzY29wZXMiOlsiemRzLnNjb3Blcy56YWtlbi5hYW5t"
+            "YWtlbiJdLCJ6YWFrdHlwZXMiOlsiaHR0cDovL3p0Yy5ubC9hcGkvdjEve"
+            "mFha3R5cGUvMTIzNCJdfX0.NHcWwoRYMuZ5IoUAWUs2lZFxLVLGhIDnU_"
+            "LWTjyGCD4",
+            "kanalen": [
+                {
+                    "naam": "zaken",
+                    "filters": {
+                        "bron": "082096752011",
+                        "zaaktype": "example.com/api/v1/zaaktypen/5aa5c",
+                        "vertrouwelijkheidaanduiding": "*",
+                    },
+                },
+                {"naam": "informatieobjecten", "filters": {"bron": "082096752011"}},
+            ],
+        }
+
+        with requests_mock.mock() as m:
+            m.register_uri(
+                "POST",
+                "https://ref.tst.vng.cloud/zrc/api/v1/callbacks",
+                status_code=204,
+            )
+            response = self.client.post(abonnement_create_url, data)
```

Comment:
I'd move this `requests_mock.Mocker()` to be a test method decorator to de-clutter the test

additionally, the preferred sugar syntax is to use `m.post(url, status_code=204)` rather than the overly verbose `m.register_uri`, to mimick the `requests` API.